lib/snippets.cson

```diff
@@ -5,13 +5,22 @@
       {
         "${1:.source.js}": {
           "${2:Snippet Name}": {
-            "prefix": "${3:hello}",
+            "prefix": "${3:Snippet Trigger}",
             "body": "${4:Hello World!}"
           }
         }
       }$5
     """
 
+  'Atom Snippet Keys':
```

Comment:
Sorry for the delay in getting back to you on this, @taylon. You asked:

> Atom Snippet With No Selector sounds good, but what would be the prefix for that?

What do you think of `snipns`?

> Instead of creating a new snippet we can make the fist line optional, so type 'snip' hit tab and it will highlight the whole line including the tabs in the next line, if you don't want the selector hit backspace and tab, if you want the selector just hit tab.

That's an interesting idea. I think I'd find two snippets more intuitive than one in this case, though.